src/elements/types/ObjectProperty.js

```diff
@@ -53,6 +54,8 @@ function readKey(children) {
         children.isNode('Identifier')
     ) {
         return children.passNode();
+    } else if (children.isToken('Identifier')) {
+        return new Identifier([children.passToken()]);
```

Comment:
Just a reference to the Babylon issue and maybe to  `elementTree` module I think